src/action/mod.rs

```diff
@@ -28,6 +28,7 @@ pub mod timeline;
 pub mod network;
 pub mod insttime;
 pub mod memsize;
+pub mod client_side_file_finder;
```

Comment:
Too long, I like module names to be one word. How about `finder`? It's not like you can have server-side one on the client. And then the "file" part can be omitted, because what else would you search for?

Reply:
Renamed to `finder` as discussed offline.